### ISSUE TITLE
Gooey/Jelly Type Gore

### ISSUE THREAD

**Issue by themonke09** [studio-minus/ppg-bugs-and-requests#1747]
Alright, yeah yeah i know i just posted something but i don't care, ANYWAY so what i mean by this is lets say, you put a shotgun to a human's head and shot them, either they would be killed with some holes in their head, or their head would just pop. what i want, is for little bits of the head, to flail around and stuff kinda like the picture (credit to www for the cool picture)
![image_2022-07-14_232307306](https://user-images.githubusercontent.com/109328435/179149950-f6bf226a-3f68-42f5-9499-f03170a289a8.png)

**Comment by themonke09** [comment]
and also, gooey blood would be cool

**Comment by PossiblyRemember** [comment]
yes, but what about the template? and this looks like it would be pretty performance heavy and seemingly hard to implement (best guess, but I wouldn't know for sure)

**Comment by DindinYT37** [comment]
At this point it's just becoming per pixel destruction...

**Comment by Drogo-man** [comment]
I like jelly it taste so good

**Comment by euuen** [comment]
看不懂，what??

**Comment by euuen** [comment]
Can speak chiness??
